packages/@sanity/components/src/portal/Sticky.js

```diff
@@ -79,10 +79,17 @@ export default class StickyPortal extends React.Component {
     if (window) {
       window.addEventListener('resize', this.handleWindowResize)
     }
+
+    // Need this until we have control over all popovers with a global controller
+    // or can pass the actual scrollContainer through to this component
+    this.intervalId = window.setInterval(() => {
+      this.moveIntoPosition()
```

Comment:
Performance wise this seems a bit crazy. What if there are several of these components mounted at the same time? It' not particularly easy to track the cascade of function calls each `moveIntoPosition()`-call causes here either. Can this be optimized in some way? E.g. by using a (passive) resize/scroll listener or something?

Reply:
The problem is that we have no access to the events that should trigger this now. The error only occurs when the sticky is mounted on component that that can't pass the scrollContainer to the current sticky. The vent that should trigger this is inside pane (in the desk-tool). Normaly on other sites we can always have a passive listener on window.scroll, but in sanity  the scroll exists inside panes, and in some cases in an popup. There is no link between the scrollContainer and the component. We use "tryFindScrollContainer" to find an scrollContainer. It could be a sollution to use a "data-attribute" on the preferred scrollContainer. Say we set "data-scrollcontainer="true" in the panes and in popover. It could possibly be better.